runtime/compiler/z/codegen/S390PrivateLinkage.cpp

```diff
@@ -1583,12 +1583,14 @@ J9::Z::PrivateLinkage::buildNoPatchingVirtualDispatchWithResolve(TR::Node *callN
       int32_t vtableOffset;
       };
    
-   ccResolveVirtualData *ccResolveVirtualDataAddress = reinterpret_cast<ccResolveVirtualData *>(cg()->allocateCodeMemory(sizeof(ccResolveVirtualData), false));
-
-   if (!ccResolveVirtualDataAddress)
+   OMR::CCData *codeCacheData = cg()->getCodeCache()->manager()->getCodeCacheData();
+   OMR::CCData::index_t index;
+   if (!(codeCacheData->put(NULL, sizeof(ccResolveVirtualData), 8, NULL, index)))
       {
       comp()->failCompilation<TR::CompilationException>("Could not allocate resovle virtual dispatch metadata");
```

Comment:
Existing typo: resovle

Reply:
Fixed in https://github.com/eclipse/openj9/compare/e6572d23ab0cf110a98cc239bea656e8aa8729ec..6d1fee3c7c3d3066a711500dc5c93851f9df074c